llvm/docs/LangRef.rst

```diff
@@ -19197,6 +19197,71 @@ the follow sequence of operations:
 
 The ``mask`` operand will apply to at least the gather and scatter operations.
 
+
+.. _int_masked_compress:
+
+'``llvm.masked.compress.*``' Intrinsics
```

Comment:
Should we mark this as experimental? Something like `llvm.experimental.vector.compress`?

Reply:
I do not know what the convention is for adding this in experimental or not. Would the flow be to have it experimental for a release or two and then make it "normal"? I don't have an opinion on this, so I'm happy to follow whatever the recommended way is.

Reply:
Sorry for the delay, yeah it looks like they're removed after about a release or two, see bfc0317153dca75137fba00b5c28758d6f720963. I think it might not be a bad idea just in case we want to change things after this PR. 

Also on the topic of naming, if this in the vector family of intrinsics should the name be updated to reflect that? E.g. `@llvm.vector.blah`